src/index.css

```diff
@@ -1,63 +1,291 @@
+  line-height: 1.5;
+  font-size: 0.875rem;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
-}
-
-.Bookstore-CMS {
-  width: 100rem;
-  height: 89.625rem;
-  padding: 9.313rem 3.75rem 7.688rem 6.25rem;
-  background-color: var(--pale-grey);
-}
-
-.panel-bg {
-  width: 87.5rem;
-  height: 5.938rem;
-  margin: 0 0 2.375rem;
-  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
-  background-color: var(--white-two);
-}
-
-.BOOKS {
-  width: 3.688rem;
-  height: 1rem;
-  margin: 1.125rem 2.563rem 0.688rem 0;
-  font-size: 0.813rem;
-  font-weight: normal;
-  font-stretch: normal;
-  font-style: normal;
-  line-height: normal;
+a {
+  text-decoration: none;
+}
+
+img {
+  width: 100%;
+  display: block;
+}
+
+h1,
+h2,
+h3,
+h4 {
+  letter-spacing: var(--spacing);
+  text-transform: capitalize;
+  line-height: 1.25;
+  margin-bottom: 0.75rem;
+}
+
+h1 {
+  font-size: 3rem;
+}
+
+h2 {
+  font-size: 2rem;
+}
+
+h3 {
+  font-size: 1.5rem;
+}
+
+h4 {
+  font-size: 0.875rem;
+}
+
+p {
+  margin-bottom: 1.25rem;
+}
+
+@media screen and (min-width: 800px) {
+  h1 {
+    font-size: 4rem;
+  }
+
+  h2 {
+    font-size: 2.5rem;
+  }
+
+  h3 {
+    font-size: 2rem;
+  }
+
+  h4 {
+    font-size: 1rem;
+  }
+
+  body {
+    font-size: 1rem;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4 {
+    line-height: 1;
+  }
+
+  .formSubmit {
+    max-width: 90%;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    align-items: center;
+    gap: 1rem;
+    margin: 1rem;
+  }
+}
+
+.btn {
+  text-transform: uppercase;
+  background: var(--clr-primary);
+  color: var(--clr-white);
+  letter-spacing: var(--spacing);
+  display: inline-block;
+  font-weight: 700;
+  transition: var(--transition);
+  font-size: 0.875rem;
+  border: none;
+  cursor: pointer;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2);
+  padding: 0.4rem 1rem;
+  margin: 0.7rem;
+}
+
+.btn:hover {
+  color: var(--clr-primary);
+  background: var(--clr-primary-light);
+}
+
+.remove-btn {
+  color: var(--clr-primary);
+  letter-spacing: var(--spacing);
+  cursor: pointer;
+  font-size: 0.85rem;
+  background: transparent;
+  border: none;
+  margin-top: 0.375rem;
+  transition: var(--transition);
+}
+
+.remove-btn:hover {
+  color: var(--clr-primary-light);
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 90%;
+  background-color: #fff;
+  margin: 1rem auto;
+}
+
+nav {
+  background: var(--clr-primary);
+  padding: 1.25rem 2rem;
+  cursor: pointer;
+  width: 100%;
+}
+
+.nav-center {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav h3 {
+  margin-bottom: 0;
+  letter-spacing: 1px;
+  color: var(--clr-white);
+}
+
+.nav-container {
+  display: block;
+  position: relative;
+}
+
+nav svg {
+  width: 40px;
+  color: var(--clr-white);
+}
+
+.navLinks {
+  display: flex;
+  align-items: center;
+  padding: 0.9rem;
```

Comment:
- [ ] Great job on adding style to your project as mentioned in the project requirements! 👏 However, your styling doesn't match the original design, and due to that it doesn't adhere to some parts of the project requirements which state that 👇👇
<img width="731" alt="Screenshot 2023-07-20 at 15 38 20" src="https://github.com/aradradev/bookstore/assets/107192076/086e4fd7-175d-4d36-b2f3-557ad4b69bde">


<hr />

`Note:` **Your styles should match exactly that of the `zeplin` design. Including `font-(color, size, weight)`, `background`, `buttons`, `input`, `spacing`, `layout`, etc. You're not to personalize any part of the design.** 


<table>
<tr>
<th align=“middle”>Replin design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="703" alt="Screenshot 2023-07-20 at 15 47 28" src="https://github.com/aradradev/bookstore/assets/107192076/11c85cc2-1c12-4242-ae3a-ced985be5eca">
</td>
<td>
<img width="1296" alt="Screenshot 2023-07-20 at 15 47 48" src="https://github.com/aradradev/bookstore/assets/107192076/d92f9a9b-af80-4009-8a33-5509267e4b73">
</td>
</tr>
</table>